trestle/utils/load_distributed.py

```diff
@@ -0,0 +1,120 @@
+# -*- mode:python; coding:utf-8 -*-
+
+# Copyright (c) 2020 IBM Corp. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Module to load distributed model."""
+
+from pathlib import Path
+from typing import Any, Dict, List, Tuple, Type, Union
+
+from trestle.core import utils
+from trestle.core.base_model import OscalBaseModel
+from trestle.utils import fs
+
+
+def load_list(filepath: Path) -> Tuple[OscalBaseModel, str, List[OscalBaseModel]]:
+    """Given path to a directory of list(array) models, load the distributed models."""
```

Comment:
> It would be great if you could provide a google docstring style 'args' and 'Returns' here as the return types are a bit vague from reading the code.
> 
> E.g. the OscalBaseModel is stripped.
Not stripped. Returned list of OscalBaseModel will be fully loaded with distributed elements.

Reply:
> Actually on second thoughts maybe this should be an internal function rather than one intended to be exposed to the end user (aka leading underscore in the name).

Yeah. I thought so too at some point. Changed to internal function